portal_ux/README.rst

```diff
@@ -0,0 +1,69 @@
+.. |company| replace:: ADHOC SA
+
+.. |company_logo| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-logo.png
+   :alt: ADHOC SA
+   :target: https://www.adhoc.com.ar
+
+.. |icon| image:: https://raw.githubusercontent.com/ingadhoc/maintainer-tools/master/resources/adhoc-icon.png
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+================
+Portal Usability
+================
+
+Several Improvements:
+
+#. Add Funcionality to allow portal for companies (parents partners), by default odoo does not show parent companies on invite to portal wizard.
```

Comment:
Cambiaría por:
#. Add functionality to allow companies (parents partners) to be portal users, by default odoo does not show parent companies on invite to portal wizard